src/bundler/bundle_v2.zig

```diff
@@ -8731,7 +8735,14 @@ pub const LinkerContext = struct {
             }
         }
 
-        // TODO: banner
+        if (newline_before_comment) {
```

Comment:
This diff should be wrapped in `if (c.options.banner.len > 0)` so that it doesn't change the output when there is no banner

Reply:
@Jarred-Sumner Fixed it, thanks so much for catching that